phys2bids/interfaces/acq.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+phys2bids interface for acqknowledge files.
+"""
+from bioread import read_file
+
+from phys2bids.physio_obj import blueprint_input
+
+
+def populate_phys_input(filename, chtrig):
```

Comment:
So `chtrig` is not really an optional parameter, here? It's possible that a user would not have supplied it since it's currently listed as "optional" in the argument parser; if it's necessary for functionality though then we should change that!

Reply:
Let's do the change for now to a required input.

Reply:
Sorry, this is actually optional but has a valid default (`0`). Still needed as a required variable or this way is okish?

Reply:
Gotcha! Okay, let's address the whole `chtrig` exists sometimes but not other times in a separate issue :+1: